inference-engine/src/vpu/graph_transformer/src/middleend/passes/sw_conv_adaptation.cpp

```diff
@@ -134,8 +134,8 @@ class ConvStage final : public StageNode {
                         std::make_shared<ConvIm2ColWeightsContent>(weights->content(), newWeightsDesc));
 
                     double im2ColBufSizeF = static_cast<double>(kernelSizeX) * kernelSizeY *
-                        output->desc().dim(Dim::W) * output->desc().dim(Dim::H) * input->desc().dim(Dim::C)
-                        + 32;
+                        output->desc().dim(Dim::W) * output->desc().dim(Dim::H) * input->desc().dim(Dim::C) * sizeof(int16_t)
+                        + 64;
```

Comment:
Maybe I'm missing something, but from this change you actually increase amount of consumed memory

Reply:
For this stage, the memory size is the same. `im2ColBufSizeF` is 2x bigger that before. But the new version of `addTempBuffer` allocates 2x less memory.

Reply:
Is it because previously descriptor were used with DataType equal to FP16, but now it is U8?

Reply:
yes, that is the reason.

Reply:
Thanks for clarification